salt/utils/gitfs.py

```diff
@@ -1611,6 +1611,11 @@ class Dulwich(GitProvider):  # pylint: disable=abstract-method
     '''
     def __init__(self, opts, remote, per_remote_defaults,
                  override_params, cache_root, role='gitfs'):
+        salt.utils.warn_until(
+            'Oxygen',
+            'Dulwich will no longer be supported for {0} beginning in the '
+            'Oxygen release of Salt.'.format(role)
```

Comment:
Oxygen should be Nitrogen here too (or you can just use `{version}`, which is handy.